app/components/UI/Stake/components/StakingBalance/StakingBalance.tsx

```diff
@@ -0,0 +1,184 @@
+import { getStakesApiResponse } from './StakingBalance.types';
+import { TokenI } from '../../../../UI/Tokens/types';
+import { getDaysAndHoursRemaining } from '../../../../../util/date';
+import { filterExitRequests } from './utils';
+import { BN } from 'ethereumjs-util';
+
+// TODO: Replace mock data when connecting to backend.
+const MOCK_STAKED_ETH_ASSET = {
+  balance: '4.9999 ETH',
+  balanceFiat: '$13,292.20',
+  name: 'Staked Ethereum',
+  symbol: 'ETH',
+} as TokenI;
+
+// TODO: Replace mock data when connecting to backend.
+const MOCK_UNSTAKING_REQUESTS: getStakesApiResponse = {
+  accounts: [
+    {
+      account: '0x0123456789abcdef0123456789abcdef01234567',
+      lifetimeRewards: '43927049303048',
+      assets: '17913326707142320',
+      exitRequests: [
+        {
+          // Unstaking
+          positionTicket: '2153260738145148336740',
+          timestamp: '1727110415000',
+          totalShares: '989278156820374',
+          withdrawalTimestamp: null,
+          exitQueueIndex: '-1',
+          // claimedAssets: null,
+          claimedAssets: '900000000000000',
+          leftShares: null,
+        },
+        // Requests below are claimable.
+        {
+          positionTicket: '515964521392314631201',
+          timestamp: '1720539311000',
+          totalShares: '99473618267007',
+          withdrawalTimestamp: '0',
+          exitQueueIndex: '57',
+          claimedAssets: '100006626507361',
+          leftShares: '0',
+        },
+        {
+          positionTicket: '515964620865932898208',
+          timestamp: '1720541495000',
+          totalShares: '99473618267007',
+          withdrawalTimestamp: '0',
+          exitQueueIndex: '57',
+          claimedAssets: '100006626507361',
+          leftShares: '0',
+        },
+        {
+          positionTicket: '516604671289934191921',
+          timestamp: '1720607327000',
+          totalShares: '1929478758729790',
+          withdrawalTimestamp: '0',
+          exitQueueIndex: '58',
+          claimedAssets: '1939870510970987',
+          leftShares: '0',
+        },
+      ],
+    },
+  ],
+  exchangeRate: '1.010838047020148468',
+};
+
+const StakingBalance = () => {
+  const { styles } = useStyles(styleSheet, {});
+
+  const networkName = useSelector(selectNetworkName);
+
+  const [isGeoBlocked] = useState(true);
+
+  const { unstakingRequests, claimableRequests } = useMemo(
+    () => filterExitRequests(MOCK_UNSTAKING_REQUESTS.accounts[0].exitRequests),
+    [],
+  );
+
+  const claimableEth = useMemo(
+    () =>
+      renderFromWei(
+        claimableRequests.reduce(
+          (acc, { withdrawalTimestamp, claimedAssets }) =>
+            !!withdrawalTimestamp &&
+            Number(withdrawalTimestamp) === 0 &&
+            !!claimedAssets
+              ? acc.add(new BN(claimedAssets))
+              : acc,
+          new BN(0),
+        ),
+      ),
+    [claimableRequests],
+  );
+
+  const hasClaimableEth = !!Number(claimableEth);
```

Comment:
Do you think maybe we can avoid using double negation for better readability ? 


```suggestion
const claimableEth = useMemo(
  () =>
    renderFromWei(
      claimableRequests.reduce(
        (acc, { withdrawalTimestamp, claimedAssets }) =>
          withdrawalTimestamp && Number(withdrawalTimestamp) === 0 &&
          claimedAssets
            ? acc.add(new BN(claimedAssets))
            : acc,
        new BN(0),
      ),
    ),
  [claimableRequests],
);

const hasClaimableEth = Number(claimableEth) > 0;
```